src/QsCompiler/LlvmBindings/LLVMSharpExtensions/LLVMMemoryBufferRefExtensions.cs

```diff
@@ -1,5 +1,7 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+// <copyright file="LLVMMemoryBufferRefExtensions.cs" company="Ubiquity.NET Contributors">
```

Comment:
oh, sorry I thought that default header was used in those files by mistake.
It really simplifies things a lot to use the Ubiquty.NET header for all files in the project (so it's great that you fine with it, thanks) because StyleCop does not support different headers in the same project and it looks like, currently, there is no way to suppress it in a single file https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1701. The only alternative I see is disabling the check for the project (then custom ` stylecop.json` for the project also could be removed).